src/plugins/data/common/search/search_source/search_source.ts

```diff
@@ -693,9 +656,7 @@ export class SearchSource {
    * serializes search source fields (which can later be passed to {@link ISearchStartSearchSource})
    */
   public getSerializedFields(recurse = false) {
-    const { filter: originalFilters, ...searchSourceFields } = omit(this.getFields(recurse), [
-      'size',
-    ]);
+    const { filter: originalFilters, ...searchSourceFields } = omit(this.getFields(), ['size']);
```

Comment:
This can be done without `lodash`.

```suggestion
    const { filter: originalFilters, size: omit, ...searchSourceFields } = this.getFields();
```